src/Features/Core/Portable/Wrapping/SeparatedSyntaxList/AbstractSeparatedSyntaxListWrapper.cs

```diff
@@ -13,24 +14,16 @@ namespace Microsoft.CodeAnalysis.Wrapping.SeparatedSyntaxList
     internal abstract partial class AbstractSeparatedSyntaxListWrapper<
         TListSyntax,
         TListItemSyntax>
-        : AbstractSyntaxWrapper
+        : AbstractSeparatedListWrapper<TListSyntax, TListItemSyntax>
```

Comment:
We should def have comments on here.  i.e. to explain the subtle difference between AbstractSeparatedSyntaxListWrapper and AbstractSeparatedListWrapper.

Reply:
I moved the original comment here to the AbstractSeparatedListWrapper and updated this one to say arguments or parameters specifically. I am open to rewording or tweaking if that isn't sufficient.